data/remnant/remnant 1 introduction.txt

```diff
@@ -3745,4 +3745,4 @@ mission "Remnant: Expanded Horizons Storms 1"
 			label fahrenheit
 			`	She looks at you quizzically for a moment, and then her display shifts to a mass of symbols. She skims briefly. "I have never even heard of such a unit. Apparently it is a very old one, though. Our records say that it had largely stopped being used by early in the 21st century. Well, I learned something new today." The display and its field of symbols vanishes as she slings her backpack over her shoulder.`
 			label stormend
-			`	"Thanks for the ride, Captain. I have to get working on this data, but if you are interested in helping some more, I hope to have more research expeditions planned soon."`
+			`	"Thanks for the ride, Captain. I have to get working on this data, but if you are interested in helping some more, I hope to have more research expeditions planned soon."`
```

Comment:
```suggestion
			`	"Thanks for the ride, Captain. I have to get working on this data, but if you are interested in helping some more, I hope to have more research expeditions planned soon."`

```

I think this change might just be picking up on the lack of a newline on the end of this file, probably removed when the missions were moved over to the other file.

Reply:
Feel free to mark this as resolved.